stack/helm/dave-admin-portal/templates/service.yaml

```
@@ -0,0 +1,23 @@
+metadata:
+  name: admin-portal
+  labels:
+    {{- include "daveAdminPortal.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.port }}
+      targetPort: http
+      protocol: TCP
+      name: http
+  ports:
```

YAMLlint (1.38.0)

[error] duplication of key "ports" in mapping

(key-duplicates)